test/history.jl

```diff
@@ -1,24 +0,0 @@
-using IterativeSolvers
-using Plots
```

Comment:
Is plotting no longer supported?

Reply:
Well, the `test/history.jl` file is basically just calling `plot(...)` on the convergence history object of all kinds of solvers. It's not testing whether it's plotting the correct thing (there are no tests in that file).

Let me just remove this commit for now and open another PR later.

Reply:
At any rate, the tests are failing on 0.7 now because of dependencies of Plots.jl, but we can't really help that.